packages/wrangler/src/__tests__/queues.test.ts

```diff
@@ -382,13 +384,14 @@ describe("wrangler", () => {
 							batch_size: 20,
 							max_retries: 3,
 							max_wait_time_ms: 10 * 1000,
+							max_concurrency: 3,
 						},
 						dead_letter_queue: "myDLQ",
 					};
 					mockPostRequest("testQueue", expectedBody);
 
 					await runWrangler(
-						"queues consumer add testQueue testScript --env myEnv --batch-size 20 --batch-timeout 10 --message-retries 3 --dead-letter-queue myDLQ"
+						"queues consumer add testQueue testScript --env myEnv --batch-size 20 --batch-timeout 10 --message-retries 3 --max-concurrency 3 --dead-letter-queue myDLQ"
```

Comment:
Can we have a test that uses `--no-max-concurrency`?

Reply:
I didn't add that flag, it wasn't needed for the`consumer add` command. For that command, the user can either pass a value (e.g. `--max-concurrency 5`) or omit it to select "no max".  I think I've got both of those cases covered in the tests.